raiden-dapp/src/components/account/Withdrawal.vue

```diff
@@ -156,15 +154,24 @@ export default class Withdrawal extends Mixins(BlockieMixin) {
   withdraw: Token | null = null;
 
   async mounted() {
-    this.balances = await this.$raiden.getRaidenAccountBalances();
-
-    if ((this.udcToken.balance as BigNumber).gt(Zero)) {
-      this.balances = uniqBy(
-        [...this.balances].concat(this.udcToken),
-        (token) => token.address
-      );
-    }
+    const balancesWithoutMainnetUtilityToken = await this.$raiden.getRaidenAccountBalances();
+    const balancesWithMainnetUtilityToken = uniqBy(
+      [...balancesWithoutMainnetUtilityToken].concat(this.udcToken),
```

Comment:
You don't need to spread the original array, it's ephemeral anyway, and also `concat` doesn't mutate the arrays.